### ISSUE TITLE
Impossible to creation an "internal" hyperlink (Place in Document hyperlink))

### ISSUE THREAD

**Issue by lredor** [ONLYOFFICE/DocumentServer#1422]
**Do you want to request a *feature* or report a *bug*?**

Bug

**What is the current behavior?**

Currently, it is not possible to create an hyperlink pointing to a title of the current document.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**

Steps to reproduce:
- Create a new document "sample.docx"
- Add this text:
```
This is a link.
Title A
Bloc A

Title B
Bloc B

Title C
Bloc C
```
- Set "Title 1" style for "Title X" lines
- Select the word "link" on the first line
- Click on "Ctrl+K"
- In the dialog, click on "Place in Document"
- KO: In the list, there is only "Beginning of document" and not all the "Title lines"

**What is the expected behavior?**

The "title" lines should be displayed as explained in documentation: https://helpcenter.onlyoffice.com/en/ONLYOFFICE-Editors/ONLYOFFICE-Document-Editor/UsageInstructions/AddHyperlinks.aspx

**Did this work in previous versions of DocumentServer?**

I'm not sure but I think.

**DocumentServer version:** Document Editor Version 6.3.1

**Operating System:** Windows 10

**Browser version:** Goolge Chrome Version 93.0.4577.63

**Comment by ShockwaveNN** [comment]
@matveevms Please take a look

**Comment by TatianaKirillova** [comment]
Hello, **@lredor**
Thanks for your request!
In an open dialog, all headers are collapsed. You must expand the "Headings" line.  ![Раскрыть список](https://user-images.githubusercontent.com/72191601/133090715-f4f5dddc-2748-43f1-a3fe-fc6b10d4b288.png)
I was unable to replicate your problem. If the problem persists, send your video, please!

**Comment by lredor** [comment]
I have no "Headings" entry. I will record a video.

**Comment by lredor** [comment]
Here is the video showing the problem :
https://user-images.githubusercontent.com/1825882/133112057-43f52f57-72ee-4568-bff8-7197de9ab2ed.mp4

**Comment by TatianaKirillova** [comment]
In our editor, such a feature of setting the style. If you select text without a paragraph mark, then the settings are applied only to the text. To expose Headings, you need to select the text along with the paragraph mark.
Try to select the text with the mouse.

**Comment by lredor** [comment]
The problem is similar with paragraph mark: https://user-images.githubusercontent.com/1825882/133212873-84b5dcda-fa44-4e26-9521-023003ec9979.mp4

**Comment by TatianaKirillova** [comment]
The problem with displaying "Headings" exists on your version of the editor 6.3.1. It has been fixed in version 6.3.2. Please update your editor to version 6.4. See [issue 1368](https://github.com/ONLYOFFICE/DocumentServer/issues/1368) for more details

**Comment by lredor** [comment]
I will ask to our system administrator. Thanks.

**Comment by ShockwaveNN** [comment]
I'll close this issue but if this problem exists after upgrade - please feel free to comment and I'll reopen this issue

**Comment by lredor** [comment]
Just a message to confirm that the problem is solved after an update to 6.4 version.
